pennylane/capture/__init__.py

```diff
@@ -147,6 +147,7 @@ class MyCustomOp(qml.operation.Operator):
     def _(*args, **kwargs):
         return type.__call__(MyCustomOp, *args, **kwargs)
 """
+from pennylane.capture import autograph
```

Comment:
Is this done so that we can import `qml.capture.autograph`? I feel like that shouldn't be necessary right?

Reply:
good point 😅